docs/podarchy-explorer/01-intro.md

```diff
@@ -0,0 +1,33 @@
+---
+id: podarchy-explorer
+title: Search and Explore
+---
+
+##### Explore on-chain structures with the Podarchy Explorer.
+---
+
+The Podarchy Explorer is a spatial interface that indexes and visualizes on-chain permissions and other relationships.
```

Comment:
We could add a sentence after this saying something like, "With the Explorer, operators can quickly diligence protocols they want to partner with and security researchers can map associated permissions of systems they are investigating"